### ISSUE TITLE
covert reducer using ngrx/entity ?

### ISSUE THREAD

**Issue by rijine** [UltimateAngular/ngrx-store-effects-app#16]

**Comment by gssjr** [comment]
I was wondering about this too:

https://github.com/ngrx/platform/tree/master/docs/entity

**Comment by tunecino** [comment]
I don't think it is a good idea. I've seen another tutorial before this one using it from start and it was confusing for me, ngrx/entity is a helper module that was introduced later to help reducing code boilerplates. I like it very much and I'm using it with most of my entities now but I also liked the fact that it wasn't used in this tutorial so I was able to better understand how ngrx/redux works. I think using helpers designed to reduces/hides parts of code should be avoided at earlier learning steps.

**Comment by rijine** [comment]
@tunecino agree...in future you might be using entity also. so at least enhancing this with entity will be another good step. May be to different branch!
